android/app/src/main/java/com/zbaymobile/Utils/Utils.kt

```diff
@@ -0,0 +1,104 @@
+package com.zbaymobile.Utils
+
+import android.content.Context
+import android.os.Build
+import android.util.Log
+import com.zbaymobile.Utils.Const.TAG_NOTICE
+import java.io.BufferedReader
+import java.io.File
+import java.io.InputStreamReader
+import java.net.ConnectException
+import java.net.InetSocketAddress
+import java.net.Socket
+
+
+object Utils {
+
+    fun getNativeLibraryDir(context: Context): String? {
+        val appInfo = context.applicationInfo
+        return appInfo.nativeLibraryDir
+    }
+
+    fun getArch(): String {
+        var folder = Build.CPU_ABI
+        val javaArch = System.getProperty("os.arch")
+        if (javaArch != null && javaArch.contains("686")) {
+            folder = "x86"
```

Comment:
probably would be fair to crash the application for now since we don't support this architecture